src/ess/spectroscopy/indirect/ki.py

```diff
@@ -35,6 +50,9 @@ def primary_spectrometer_coordinate_transformation_graph() -> (
         {
             "incident_beam": straight_incident_beam,
             "L1": L1,
+            "sample_position": lambda: sample_position,
+            "source_position": lambda: source_position,
```

Comment:
Does having L1 imply the source position is actually unused? Or might a bug be hiding somewhere?

Reply:
Yes. Bad naming scheme in SCippNeutron...